classes/ImageManager.php

```diff
@@ -688,6 +688,12 @@ public static function write($type, $resource, $filename)
 
                 break;
 
+            case 'avif':
+                // @phpstan-ignore-next-line
+                $success = imageavif($resource, $filename, Configuration::get('PS_ADDITIONAL_IMAGE_AVIF_QUALITY'));
```

Comment:
Maybe we should unify the quality logic with the other ones and provide a fallback?